node/db/src/rocks.rs

```diff
@@ -76,11 +77,17 @@ impl RocksDb {
     where
         P: AsRef<Path>,
     {
-        let db_opts = Self::to_options(config);
+        let mut db_opts = Self::to_options(config);
+        db_opts.enable_statistics();
```

Comment:
We can already enable rocksdb statistics using the toml config file:

```
[rocks_db]
create_if_missing = true
parallelism = 12
write_buffer_size = 268435456
max_open_files = 1024
compaction_style = "level"
compression_type = "lz4"
enable_statistics = false
...
```

So I would remove the call to `enable_statistics()`.

Reply:
Ah i c. Should we enable it by default?

Reply:
If there is not too much performance overhead maybe?

Reply:
I cannot find it in doc, but I don't observe notable performance / mem usage different locally when running mainnet

Reply:
> The overhead of statistics is usually small but non-negligible. We usually observe an overhead of 5%-10%.

See here: https://github.com/facebook/rocksdb/wiki/Statistics#stats-level-and-performance-costs

Reply:
Thanks for the info, let me turn it off by default